site/en/docs/privacy-sandbox/overview/index.md

```diff
@@ -119,12 +119,38 @@ without passive tracking.
 
 ## Who is working on the Privacy Sandbox?
 
-By early 2021 there were:
-* 30+ Privacy Sandbox proposals offered by Chrome and others.
-* 400+ participants who joined W3C groups to provide input including the
+Chrome and other browser vendors, as well as ad companies and other
+stakeholders, have offered more than 30 proposals to date. These proposals can
+be found in the
+[public resources of W3C groups](https://github.com/w3c/web-advertising#ideas-and-proposals-links-outside-this-repo)
+and cover a wide variety of use cases and requirements.
+
+400+ participants have joined W3C groups to provide input including the
 [Improving Web Advertising Business Group](https://www.w3.org/community/web-adv/participants) and
 the [Privacy Community Group](https://www.w3.org/community/privacycg/participants).
-* Five API implementations available for testing in Chrome.
+Five API implementations are currently available for testing in Chrome.
```

Comment:
```suggestion
## Where are the Privacy Sandbox APIs available?

Five API implementations are currently available for testing in Chrome.
```

Although this was in the "Who is working on Chrome" section, it really deserves its own heading about implementation.